src/common/DeveloperConfig.ts

```diff
@@ -113,6 +113,9 @@ export type DeveloperConfig = {
     HomeStack: Navigator<ParamList, NativeStackNavigatorProps>,
   ) => JSX.Element[];
   CustomHomeScreen?: () => JSX.Element;
+  CustomActivityIndicatorView?: React.FC<{
+    timeoutMessage: JSX.Element | null;
```

Comment:
Seems okay but it might be better to just pass the message through as a string and let the custom view decide what to do with it.

Reply:
I suppose it might be difficult to write your own timeout message component if you don't have access to the message. I'll follow up in a different PR.